src/lightning_app/__init__.py

```diff
@@ -19,8 +19,13 @@
     _logger.propagate = False
 
 
+from lightning_app import __about__  # noqa: E402
 from lightning_app import components  # noqa: E402, F401
 from lightning_app.__about__ import *  # noqa: E402, F401, F403
+
+if not hasattr(__about__, "__version__"):
```

Comment:
Is it to resolve a bug with the meta package generator ?

Reply:
Not bug, but compatibility as in future we move version to separate file